composer.json

```diff
@@ -42,7 +42,8 @@
         "behat/symfony2-extension": "^2.1",
         "doctrine/doctrine-fixtures-bundle": "^2.3",
         "sensio/generator-bundle": "^3.0",
-        "symfony/phpunit-bridge": "^3.0"
+        "symfony/phpunit-bridge": "^3.0",
+        "phing/phing": "2.*"
```

Comment:
install phing in Dockerfile, not like this. otherwise phing requires composer which requires phing and it's all broken.